### ISSUE TITLE
Limit amount of 2Catpcha trials?

### ISSUE THREAD

**Issue by walidproggen** [flathunters/flathunter#222]
Hello together,

currently i have the problem that flathuner is making too much requests for 2Captcha.
I had days where the amount of requests exceeded 1000, in a single day.
So i lost very quick money. 

Im aware of the config-option "sleeping_time". But increasing this will only make me slow for flathunting.
 It would be good to have an option like "max_requests_per_hour" or something like that for 2Captcha.

**Comment by codders** [comment]
Hi there,

This makes sense. We should certainly avoid that we burn through captchas so fast. Would you be interested and able to make a pull request for this?

**Comment by flo-wolf** [comment]
How many captchas do you guys have to solve? My tests came out to be exactly one every hour, no matter the delays.

**Comment by flo-wolf** [comment]
> I don't know exactly how fast they're coming, but I know that $10 dollars at 2-captcha lasts more than a month for a server running all the time (but sleeping between 23h and 6h).

Yup that sounds about right. Exactly one an hour equals to 720 a month, and at 1000 captchas under 3€ its really not that expensive. 

Anyways, I scraped for 4 days straight with a 30sec delay between reloads, and then another 4 days with 5 minutes. In both cases captchas came always at the first reload after one hour has passed - i tracked it and calculated averages, which came out slightly over an hour every time, never less
